source/matplot/core/axis_type.h

```diff
@@ -106,6 +107,9 @@ namespace matplot {
             return *this;
         }
 
+        bool is_timestamp() const;
```

Comment:
Is the timestamp related to the `ss.precision(5)`?

Reply:
`timestamp(true)` enables timestamp rendering, so `ax->xtickformat ("%m/%d/%Y %H:%M:%S");` can be used for example. `ss.precision()` is relevant to large numbers